### DISCUSSION TITLE
Invite mails not sent

### DISCUSSION THREAD

**Discussion by FromNestToNest** [supabase/supabase#31051] [Questions] [answered]
We are having issues when inviting users from the supabase dashboard, they will never receive an email with a confirmation link, thus preventing any user to create their account (we are an invite only app)

Magic links seems unaffected and the mails are sent correctly...

Thanks for the help

**Comment by GaryAustin1** [comment]
Any errors in the auth or Postgres logs?

**Comment by pikanezi** [comment]
No, no error at all:
![image](https://github.com/user-attachments/assets/1efd2bbe-968a-417d-840e-efdf13c8dde3)

**Comment by FromNestToNest** [comment, marked as answer]
We were using this emoji in the title of the mail template : `✉️ You are invited to join From Nest To Nest!` and it was the culprit, using another emoji worked, but this one broke the mails for no obvious reason 😅

Using `🏠 You are invited to join From Nest To Nest!` works

✉️✉️✉️
